nixos/tests/pppd.nix

```diff
@@ -0,0 +1,62 @@
+import ./make-test.nix (
+  let
+    chap-secrets = {
+      text = ''"flynn" * "reindeerflotilla" *'';
```

Comment:
I assume this is not a real password :-)

Reply:
You assume correctly :) It's Flynn's password from [Tron](https://en.wikipedia.org/wiki/Tron). It's a classic "I need a quick password for testing" :P